### PR TITLE
OCPBUGS#30031: Doc improvements related to "Monitoring" section

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.12+

Issue:
[OCPBUGS-30031](https://issues.redhat.com/browse/OCPBUGS-30031)

Link to docs preview:
[Preview](https://72264--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-monitoring-using-lvms_logical-volume-manager-storage)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@jakobmoellerdev , @suleymanakbas91 
Can you please review this PR?
Thanks.

File: modules/lvms-download-log-files-and-diagnostics.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 When {lvms} is unable to automatically resolve a problem, use the must-gather tool to collect the log files and diagnostic information so that you or the Red Hat Support can review the problem and determine a solution.
 
-* Run the must-gather command from the client connected to {lvms} cluster by running the following command:
+* Run the must-gather command from the client connected to {lvms} cluster:
```
Comment (dfitzmau) [inline modules/lvms-download-log-files-and-diagnostics.adoc:11]:
```suggestion
* Run the `must-gather` command from the client connected to {lvms} cluster:
```

[Example](https://docs.openshift.com/container-platform/4.15/nodes/cma/nodes-cma-autoscaling-custom-debugging.html) of highlighting in other docs.

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc [hunk 2]
```diff
@@ -6,42 +6,56 @@
 [id="lvms-monitoring-using-lvms_{context}"]
 = Monitoring {lvms}
 
-When you use {rh-rhacm} to install {lvms}, you must configure {rh-rhacm} Observability to monitor all the clusters from one place.
+To enable cluster monitoring, you must add the following label in the namespace where you have installed {lvms}:
+[source,text]
+----
+openshift.io/cluster-monitoring=true
```
Comment (dfitzmau) [inline modules/lvms-monitoring-logical-volume-manager-operator.adoc:12]: I'm not fully sure on the context of where to apply this command. Add to a config file, assign to an environment variable, or launch a tool and specify the command there?

Examples from the Adding custom metrics link:

Screenshot from 2024-03-08 10-24-48
Screenshot from 2024-03-08 10-24-51

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc [hunk 3]
```diff
@@ -6,42 +6,56 @@
...
-[source,terminal]
-----
-topolvm_thinpool_data_percent
-topolvm_thinpool_metadata_percent
-topolvm_thinpool_size_bytes
-----
+The following table describes the `topolvm` metrics:
+
+.`topolvm` metrics
+[%autowidth,frame="topbot",options="header"]
```
Comment (dfitzmau) [inline modules/lvms-monitoring-logical-volume-manager-operator.adoc:28]: I like this table. I know "topbot" is valid a HTML component but [Asciidoctor](https://docs.asciidoctor.org/asciidoc/latest/tables/borders/) does not list it. Or platforms can be a bit funny when it comes to supporting cetain syntax. I do not want to kill your formatting buzz but asking a merge reviewer to check how platform docs render on Pantheon would be worthwile.

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc [hunk 4]
```diff
@@ -6,42 +6,56 @@
...
 ====
 
 [id="lvms-monitoring-using-lvms-alerts_{context}"]
 == Alerts
 
-When the thin pool and volume group are filled up, further operations fail and might lead to data loss.
-{lvms} sends the following alerts about the usage of the thin pool and volume group when utilization crosses a certain value:
+When the thin pool and volume group reach maximum storage capacity, further operations fail. This can lead to data loss.
+
+{lvms} sends the following alerts when the utilization of the thin pool and volume group crosses a certain value:
```
Comment (dfitzmau) [inline modules/lvms-monitoring-logical-volume-manager-operator.adoc:49]: Screenshot from 2024-03-08 10-32-21

Comment (dfitzmau) [inline modules/lvms-monitoring-logical-volume-manager-operator.adoc:49]:
```suggestion
{lvms} sends the following alerts when the usage of the thin pool and volume group exceeds a certain value:
```

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc [hunk 5]
```diff
@@ -6,42 +6,56 @@
...
-|`VolumeGroupUsageAtThresholdCritical`|This alert is triggered when both the volume group and thin pool utilization cross 85% on nodes. `VolumeGroup` is critically full. Data deletion or volume group expansion is required.
+|`VolumeGroupUsageAtThresholdCritical`|This alert is triggered when both the volume group and thin pool utilization cross 85% on nodes. In this case, the volume group is critically full. Data deletion or volume group expansion is required.
 |`ThinPoolDataUsageAtThresholdNearFull`|This alert is triggered when the thin pool data utilization in the volume group crosses 75% on nodes. Data deletion or thin pool expansion is required.
-|`ThinPoolDataUsageAtThresholdCritical`|This alert is triggered when  the thin pool data utilization in the volume group crosses 85% on nodes. Data deletion or thin pool expansion is required.
+|`ThinPoolDataUsageAtThresholdCritical`|This alert is triggered when the thin pool data utilization in the volume group crosses 85% on nodes. Data deletion or thin pool expansion is required.
```
Comment (dfitzmau) [inline modules/lvms-monitoring-logical-volume-manager-operator.adoc:58]:
```suggestion
|`ThinPoolDataUsageAtThresholdCritical`|This alert is triggered when the thin pool data usage in the volume group exceeds 85% on nodes. Data deletion or thin pool expansion is required.
```

File: modules/lvms-download-log-files-and-diagnostics.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/lvms-download-log-files-and-diagnostics.adoc:11]: Updated as per the suggestion.

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc (hunk 4 above)
Comment (sr1kar99) [reply modules/lvms-monitoring-logical-volume-manager-operator.adoc:49]: Updated as per the suggestion.

Comment (sr1kar99) [reply modules/lvms-monitoring-logical-volume-manager-operator.adoc:49]: Updated as per the suggestion.

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc (hunk 5 above)
Comment (sr1kar99) [reply modules/lvms-monitoring-logical-volume-manager-operator.adoc:58]: Updated as per the suggestion.

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc (hunk 2 above)
Comment (sr1kar99) [reply modules/lvms-monitoring-logical-volume-manager-operator.adoc:12]: This label must be added in a namespace:

Screenshot from 2024-03-08 16-43-01

Click *Edit* to add a new label.

**Comment by sr1kar99** [comment]
> Hi @sr1kar99 . Excellent work to you and the engineers. Discussing the limitations when limtis are reached is a tough artform. I added comments inline. I like your "topbot" styling use but I'm not sure how our platform infrastructure will handle it come publication.
> 
> Let me know if you would like to present the styling to the wider team and get their viewpoint.
> 
> /remove-label peer-review-in-progress
> 
> /label peer-review-done

Hi @dfitzmau 
Thank you for the review. 
Regarding the topbot frame, I was trying to explore the styling options, but not sure if we can use it as I though it could lead to inconsistent styling of tables. 
Do we have guidelines on this?

**Comment by dfitzmau** [comment]
> > Hi @sr1kar99 . Excellent work to you and the engineers. Discussing the limitations when limits are reached is a tough artform. I added comments inline. I like your "topbot" styling use but I'm not sure how our platform infrastructure will handle it come publication.
> > Let me know if you would like to present the styling to the wider team and get their viewpoint.
> > /remove-label peer-review-in-progress
> > /label peer-review-done
> 
> Hi @dfitzmau Thank you for the review. Regarding the topbot frame, I was trying to explore the styling options, but not sure if we can use it as I thought it could lead to inconsistent styling of tables. Do we have guidelines on this?

Hi @sr1kar99 . Tbale gudiance is lacking in our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting) doc. Usually when we something listed in the Asciidoc documentation, we can experiment. I'll add something to the team docs channel.

File: modules/lvms-monitoring-logical-volume-manager-operator.adoc (hunk 2 above)
Comment (dfitzmau) [reply modules/lvms-monitoring-logical-volume-manager-operator.adoc:12]: Ah, OK. Thanks. This makes sense.

**Comment by dfitzmau** [comment]
> > > Hi @sr1kar99 . Excellent work to you and the engineers. Discussing the limitations when limits are reached is a tough artform. I added comments inline. I like your "topbot" styling use but I'm not sure how our platform infrastructure will handle it come publication.
> > > Let me know if you would like to present the styling to the wider team and get their viewpoint.
> > > /remove-label peer-review-in-progress
> > > /label peer-review-done
> > 
> > 
> > Hi @dfitzmau Thank you for the review. Regarding the topbot frame, I was trying to explore the styling options, but not sure if we can use it as I thought it could lead to inconsistent styling of tables. Do we have guidelines on this?
> 
> Hi @sr1kar99 . Tbale gudiance is lacking in our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting) doc. Usually when we something listed in the Asciidoc documentation, we can experiment. I'll add something to the team docs channel.

I've asked [here](https://redhat-internal.slack.com/archives/C02KBD8A4UF/p1709897478675109).

**Comment by sr1kar99** [comment]
> > > > Hi @sr1kar99 . Excellent work to you and the engineers. Discussing the limitations when limits are reached is a tough artform. I added comments inline. I like your "topbot" styling use but I'm not sure how our platform infrastructure will handle it come publication.
> > > > Let me know if you would like to present the styling to the wider team and get their viewpoint.
> > > > /remove-label peer-review-in-progress
> > > > /label peer-review-done
> > > 
> > > 
> > > Hi @dfitzmau Thank you for the review. Regarding the topbot frame, I was trying to explore the styling options, but not sure if we can use it as I thought it could lead to inconsistent styling of tables. Do we have guidelines on this?
> > 
> > 
> > Hi @sr1kar99 . Tbale gudiance is lacking in our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting) doc. Usually when we something listed in the Asciidoc documentation, we can experiment. I'll add something to the team docs channel.
> 
> I've asked [here](https://redhat-internal.slack.com/archives/C02KBD8A4UF/p1709897478675109).

Hi @dfitzmau 
Thanks very much for adding this query to the docs channel. 
As discussed, I have removed the frame option.

**Comment by dfitzmau** [comment]
> > > > > Hi @sr1kar99 . Excellent work to you and the engineers. Discussing the limitations when limits are reached is a tough artform. I added comments inline. I like your "topbot" styling use but I'm not sure how our platform infrastructure will handle it come publication.
> > > > > Let me know if you would like to present the styling to the wider team and get their viewpoint.
> > > > > /remove-label peer-review-in-progress
> > > > > /label peer-review-done
> > > > 
> > > > 
> > > > Hi @dfitzmau Thank you for the review. Regarding the topbot frame, I was trying to explore the styling options, but not sure if we can use it as I thought it could lead to inconsistent styling of tables. Do we have guidelines on this?
> > > 
> > > 
> > > Hi @sr1kar99 . Tbale gudiance is lacking in our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting) doc. Usually when we something listed in the Asciidoc documentation, we can experiment. I'll add something to the team docs channel.
> > 
> > 
> > I've asked [here](https://redhat-internal.slack.com/archives/C02KBD8A4UF/p1709897478675109).
> 
> Hi @dfitzmau Thanks very much for adding this query to the docs channel. As discussed, I have removed the frame option.

Thanks, @sr1kar99 . Sorry to be _Sargeant Buzzkill_ but sometimes our tooling really limits things. LGTM!

**Comment by sr1kar99** [comment]
> > > > > > Hi @sr1kar99 . Excellent work to you and the engineers. Discussing the limitations when limits are reached is a tough artform. I added comments inline. I like your "topbot" styling use but I'm not sure how our platform infrastructure will handle it come publication.
> > > > > > Let me know if you would like to present the styling to the wider team and get their viewpoint.
> > > > > > /remove-label peer-review-in-progress
> > > > > > /label peer-review-done
> > > > > 
> > > > > 
> > > > > Hi @dfitzmau Thank you for the review. Regarding the topbot frame, I was trying to explore the styling options, but not sure if we can use it as I thought it could lead to inconsistent styling of tables. Do we have guidelines on this?
> > > > 
> > > > 
> > > > Hi @sr1kar99 . Tbale gudiance is lacking in our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting) doc. Usually when we something listed in the Asciidoc documentation, we can experiment. I'll add something to the team docs channel.
> > > 
> > > 
> > > I've asked [here](https://redhat-internal.slack.com/archives/C02KBD8A4UF/p1709897478675109).
> > 
> > 
> > Hi @dfitzmau Thanks very much for adding this query to the docs channel. As discussed, I have removed the frame option.
> 
> Thanks, @sr1kar99 . Sorry to be _Sargeant Buzzkill_ but sometimes our tooling really limits things. LGTM!

@dfitzmau haha! NP, Darragh! Thank you for shedding light on it. :)

**Comment by sr1kar99** [comment]
Hi @jakobmoellerdev @suleymanakbas91 
All the reviews for this PR are done. Before proceeding with merge review, I wanted to confirm if we can merge to 4.12+
The current documentation for Monitoring seems to be unchanged since 4.12. 
Can we apply the updates in this PR to all the branches (4.12+)..?

File: modules/lvms-download-log-files-and-diagnostics.adoc [hunk 6]
```diff
@@ -8,7 +8,7 @@
 
 When {lvms} is unable to automatically resolve a problem, use the must-gather tool to collect the log files and diagnostic information so that you or the Red Hat Support can review the problem and determine a solution.
 
-* Run the must-gather command from the client connected to {lvms} cluster by running the following command:
+* Run the `must-gather` command from the client connected to {lvms} cluster:
 +
 [source,terminal,subs="attributes+"]
```
Comment (adellape) [inline modules/lvms-download-log-files-and-diagnostics.adoc:13]:
```suggestion
.Procedure
* Run the `must-gather` command from the client connected to {lvms} cluster:
+
[source,terminal,subs="attributes+"]
```

**Review by adellape — COMMENTED** [review]
Would like to hold merge on this to get a `.Procedure` added in.

**Comment by adellape** [comment]
Please re-queue for merge-review again when ready.

File: modules/lvms-download-log-files-and-diagnostics.adoc (hunk 6 above)
Comment (sr1kar99) [reply modules/lvms-download-log-files-and-diagnostics.adoc]: @adellape 
Updated as per the suggestion.

**Comment by ShaunaDiaz** [comment]
@sr1kar99 1) To my knowledge, 4.12 is EOL and we are not updating those docs. 2) You can make the manual cherry picks for 4.13 and 4.14, resolve the conflicts, and either let me know, or put them back in the merge review queue and we can take care of those for you.

**Comment by sr1kar99** [comment]
@ShaunaDiaz I have manually cherry-picked to 4.12,  4.13, and 4.14, 4.15, and 4.16. Could you PTAL and merge the PRs?
Thanks.

**Pull request merged**